arctic/tickstore/tickstore.py

```diff
@@ -343,7 +343,7 @@ def read(self, symbol, date_range=None, columns=None, include_images=False, allo
             rtn = rtn.sort_index(kind='mergesort')
         if date_range:
             # FIXME: support DateRange.interval...
-            rtn = rtn.ix[date_range.start:date_range.end]
+            rtn = rtn[date_range.start:date_range.end]
```

Comment:
it could be iloc, but start and end are dates (datetime, timestamp, doesnt matter which) so slicing works as well. Note I also made the same change in arctic_copy_data.py in scripts. Happy to change it to iloc if you think thats preferable or if there is some problem with datetime slicing I'm not aware of.